<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Database</class>
 <widget class="QWidget" name="Database">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #83A2FF;
	color: #FFFFFF;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <property name="bottomMargin">
      <number>50</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2">
       <item>
        <widget class="QLabel" name="title_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 50%; /* Makes the label circular */
border: 2px solid #000000; /* Border color and width */
padding: 10px; /* Adjust padding as needed */
background-color: #83A2FF;
color: #FFD28F;
border: none;
</string>
         </property>
         <property name="text">
          <string>Database</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>100</number>
         </property>
         <property name="topMargin">
          <number>30</number>
         </property>
         <property name="rightMargin">
          <number>100</number>
         </property>
         <property name="bottomMargin">
          <number>30</number>
         </property>
         <item>
          <widget class="QPushButton" name="go_back_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	border-radius: 20px; /* Adjust the radius as needed */
	background-color: #FFE3BB;
}
QPushButton:hover {
	background-color: #FFD28F;
	}
QPushButton:pressed {
	background-color: #B4BDFF;
}</string>
           </property>
           <property name="text">
            <string>Go back</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="summary_table">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableView {
     color: white;     /* Set text color for cells */
	border: 2px solid #000000; /* Border color and width */
	padding: 10px; /* Adjust padding as needed */
	background-color: #FFE3BB;
	color: black;
	font-weight: bold;
	border: none;
}

QHeaderView::section {
     color: white;     /* Set text color for header */
     background-color: #FFD28F; /* Set background color for header */
     border: 1px solid #555; /* Set border for header */
	height: 36px;     /* Set height for each header */
	width: 260px;
	font-weight: bold;
}</string>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
